opentech/apply/funds/forms.py

```diff
@@ -68,7 +68,10 @@ def __init__(self, *args, **kwargs):
         self.request = kwargs.pop('request', None)
         super().__init__(*args, **kwargs)
 
+        # All submitted reviews by non-role reviewers
         self.submitted_reviewers = User.objects.filter(id__in=self.instance.reviews.values('author'))
+        self.submitted_reviewers = self.submitted_reviewers.exclude(
+            id__in=AssignedReviewers.objects.role_reviewers_by_submission(self.instance).values('reviewer'))
```

Comment:
There are two cases:
1. I leave a review and i don't want to be added again
2. I get removed from a role, but i've already submitted a review - i dont want to be removed.

Not sure if the above handles 2?